packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -480,6 +480,26 @@
             "name": "stepfunctions_previewstatemachine",
             "description": ""
         },
+        {
+            "name": "stepfunctions_openWorkflowStudio",
+            "description": "Called after opening asl file in Step Functions Workflow Studio",
+            "metadata": [
+                {
+                    "type": "id",
+                    "required": true
+                }
+            ]
+        },
+        {
+            "name": "stepfunctions_cancelOpenWorkflowStudio",
+            "description": "Called after cancelling opening asl file in Step Functions Workflow Studio",
```

Comment:
It's never necessary to have a separate cancel event. All events can have result=success/fail/cancel. 

Please review https://github.com/aws/aws-toolkit-vscode/blob/master/docs/telemetry.md